### PR TITLE
docs: update for OEP-58 implementation - FC-55 and FC-12

### PR DESCRIPTION
**Pull request by OmarIthawi**
Docs update after OEP-58 has been implemented. 

See: 
 - https://github.com/openedx/open-edx-proposals/pull/610

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @OmarIthawi!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.



#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@feanil`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

**Review by sarina — APPROVED** [review]
I haven't tested these instructions, but they're clear and easy to read

**Comment by sarina** [comment]
@brian-smith-tcril would you take a quick read through and merge if you're satisfied?

**Comment by OmarIthawi** [comment]
@brian-smith-tcril @sarina would you mind triaging this PR?

File: source/developers/how-tos/enable-translations-new-repo.rst
Comment (sarina) [inline source/developers/how-tos/enable-translations-new-repo.rst:265]: Should this be a tag?

File: source/developers/how-tos/enable-translations-new-repo.rst
Comment (sarina) [inline source/developers/how-tos/enable-translations-new-repo.rst:187]: Is this a brand that people will have, or should this be like,

```suggestion
   - **Use workflow from your branch:** ``your-branch-name``
```

Also at line 189

File: source/developers/how-tos/enable-translations-new-repo.rst
Comment (sarina) [inline source/developers/how-tos/enable-translations-new-repo.rst:205]: Depending on the above comment should this be


```suggestion
#. Temporarily pull translations from the fork using the Makefile command e.g. ``make ATLAS_OPTIONS="--repository=Zeit-Labs/openedx-translations --revision=your-branch-name" pull_translations``
```

Comment (brian-smith-tcril) [reply source/developers/how-tos/enable-translations-new-repo.rst:205]: `--repository=Zeit-Labs/openedx-translations` should probably also be updated to be `your-branch-name`

File: source/developers/how-tos/enable-translations-new-repo.rst
Comment (brian-smith-tcril) [reply source/developers/how-tos/enable-translations-new-repo.rst:265]: I think the naming is just confusing here. The link is to the commit `feat: \`atlas pull\` for XBlock translations | FC-0012`. Linking to the tagged version wouldn't provide the same context as linking to this commit.

I think another possible option would be to link to the PR (https://github.com/openedx/edx-platform/pull/33698), so up on line 48 where it says "Redwood or newer" we could change that to be "A version of edx-platform that includes PR 33698 (Redwood or newer)."

Comment (OmarIthawi) [reply source/developers/how-tos/enable-translations-new-repo.rst]: Fixed, now Redwood is behind us. Removing the link altogether makes more sense.

**Review by OmarIthawi — COMMENTED** [review]
Thanks @brian-smith-tcril @sarina for the review and sorry for the delay. Now it's ready with all notes addressed.

**Review by brian-smith-tcril — APPROVED** [review]
LGTM!

**Review by sarina — APPROVED** [review]
I'll merge once the devstack ref is removed

File: source/developers/how-tos/enable-translations-new-repo.rst
Comment (OmarIthawi) [reply source/developers/how-tos/enable-translations-new-repo.rst]: good catch, now done!

**Pull request merged**
